### PR TITLE
Release/3.1.0

### PR DESCRIPTION
**Pull request by stefaniacardenas**
### Added
- Public: Added Proof of address `poa` step where users can capture their proof of address documents. This is a beta feature.
- Internal: Further device metadata submitted to Onfido API
- Internal: Upload fallback for the `face` step can be disabled by using the option `{ uploadFallback: false }`. The default value is `true`
- Internal: Added multi-frame capture for the `standard` variant of the face step (only for camera capture).

### Changed
- Internal: Cross device client can now only be opened on mobile browsers. Users trying to open the link on a desktop browsers will see an error.
- Internal: Removed unused development dependencies which had known vulnerabilities

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-540-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-540-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://540-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-540-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-540-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://540-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-540-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-540-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-540-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://540-pr-onfido-sdk-ui-onfido.surge.sh

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "onfido-sdk-ui",
-  "version": "3.0.1",
+  "version": "3.1.0-rc.4",
```
Comment (rfreitas) [inline package.json]: should it still be rc4?

Comment (stefaniacardenas) [reply package.json]: Oh I can see that the push of my latest commit failed. I will push it on Monday from my other laptop

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-540-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://540-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-540-pr-onfido-sdk-ui-onfido.surge.sh

File: package.json [hunk 2]
```diff
@@ -37,7 +38,8 @@
   "author": "Lawrence Gosset <lawrence.gosset@onfido.com> (https://github.com/gosseti)",
   "devDependencies": {
     "@babel/cli": "^7.0.0",
-    "@babel/core": "^7.0.0",
+    "@babel/core": "^7.1.2",
+    "babel-core": "^7.0.0-bridge.0",
```
Comment (rfreitas) [inline package.json]: @stefaniacardenas do you know on which PR this was added? I would like to take a look a little more on the context of this change

Comment (stefaniacardenas) [reply package.json]: The one that introduced Jest https://github.com/onfido/onfido-sdk-ui/pull/483/files

**Pull request merged**